src/app/(spaces)/Space.tsx

```
@@ -78,99 +77,86 @@ export default function Space({
+    const cleanedFidgetInstanceDatums = { ...config.fidgetInstanceDatums };
+    removedFidgetIds.forEach(id => {
+      delete cleanedFidgetInstanceDatums[id];
+    });
+    let settingsChanged = false;
+    // Check and rename 'fidget Shadow' to 'fidgetShadow' in each fidget's config settings
+    Object.keys(cleanedFidgetInstanceDatums).forEach((id) => {
+      const datum = cleanedFidgetInstanceDatums[id];
+      const settings = datum.config?.settings as Record<string, unknown>;
+      if (settings && "fidget Shadow" in settings) {
+        settings.fidgetShadow = settings["fidget Shadow"];
+        delete settings["fidget Shadow"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)